k8s/openebs-pre-release-features.yaml

```diff
@@ -1510,6 +1514,24 @@ spec:
 ---
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
+metadata:
+  name: jiva-volume-isvalidversion-default-0.7.0
+  namespace: openebs
+spec:
+  meta: |
+    id: is070casvolume
+    runNamespace: {{.Volume.runNamespace}}
+    apiVersion: v1
+    kind: Service
+    action: list
+    options: |-
+      labelSelector: vsm={{ .Volume.owner }}
+  post: |
+    {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "is070casvolume.name" .TaskResult | noop -}}
+    {{- .TaskResult.is070casvolume.name | empty | not | versionMismatchErr "is not a cas volume of 0.7.0 version" | saveIf "is070casvolume.versionMismatchErr" .TaskResult | noop -}}
```

Comment:
Here as well ... `.taskResult.is070jivavolume.name`

The error message should be changed `"is not a jiva volume of 0.7.0 version"`

Then ... `saveIf "is070jivavolume.versionMismatchErr" .TaskResult`